website/docs/authenticating_via_msi.html.markdown

```diff
@@ -7,17 +7,17 @@ description: |-
 
 ---
 
-# Authenticating to Azure Resource Manager using Managed Service Identity
+# Azure Provider: Authenticating using Managed Service Identity
 
-Terraform supports authenticating to Azure through Managed Service Identity, Service Principal or the Azure CLI.
+Terraform supports authenticating to Azure through Managed Service Identity, a Service Principal, or the Azure CLI.
```

Comment:
in retrospect I think this'd read clearer with the word "using" - what do you think?

```suggestion
Terraform supports authenticating to Azure using Managed Service Identity, a Service Principal, or the Azure CLI.
```

Reply:
Agreed and so changed!